lib/core/canvas_buffer.dart

```diff
@@ -33,10 +33,10 @@ class CanvasBuffer {
   int height;
 
   /// The original horizontal cursor offset in the terminal.
-  int cursorOriginalX = -1;
+  int cursorOriginalX = 1;
```

Comment:
Consider using line and column as names instead of x and y.

(It also seems that lines and columns in most editors are 1-based instead of 0-based, which makes sense with these defaults. Even though 1-based feels really weird to me.)